appium-testng-examples/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>demo-java</artifactId>
+        <groupId>com.saucelabs</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>appium-testng-examples</artifactId>
+
+    <properties>
+        <testngXmlDir>src/test/resources</testngXmlDir>
+        <feature>image_injection</feature>
+        <testngXmlFile>image_injection_android_test.xml</testngXmlFile>
+        <region>eu</region>
+        <testng.version>6.14.3</testng.version>
+        <selenium.version>3.141.59</selenium.version>
+        <maven-failsafe-plugin.version>2.19.1</maven-failsafe-plugin.version>
+        <commons-logging.version>1.2</commons-logging.version>
+        <sauce.version>1.0.44</sauce.version>
+        <cucumber.version>5.6.0</cucumber.version>
+        <sauce.version>1.0.44</sauce.version>
+        <appium.version>7.2.0</appium.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+        </dependency>
+
+        <!-- Selenium -->
+        <dependency>
```

Comment:
@eyaly don't need Selenium dependency as those are located in our repository [level POM.xml](https://github.com/saucelabs-training/demo-java/blob/d730b3a7bb565793b671cdbd80d4ecc40ea4502e/pom.xml#L54). The nice thing about this setup is that we just reuse those dependencies :) Same goes for Appium and maven-compiler-plugin

Reply:
Thanks @nadvolod for the feedback. I finished with all the changes and the fix according to your suggestions. Please review and merge